### PR TITLE
UI Improvisations

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
@championswimmer bhaiya can you give me youtube API key, for the time being, I've used my own API key

**Comment by aggarwalpulkit596** [comment]
@championswimmer  Bhaiya can you look at the travis error so that i can merge this one and start working on ExoPlayer as Harshit bhaiya told me that he'll be providing me the db structure in a day or two
screenshot 2018-11-03 at 3 41 31 pm

**Comment by the-dagger** [comment]
@aggarwalpulkit596 I think this is an issue with travis as it's unable to find the bundled library.
Also for now, we can go ahead and use a test token, and once we are ready for production, we ca replace it with CB's token.

Also, I'm working on the DB structure, will share it by today evening/late night

File: app/build.gradle [hunk 1]
```diff
@@ -63,4 +65,5 @@ dependencies {
         implementation "org.jetbrains.anko:anko-$name:$ankoVersion"
     }
     implementation project(':onlineapi')
+    implementation files('build/libs/YouTubeAndroidPlayerApi.jar')
```
Comment (the-dagger) [inline app/build.gradle]: You sure this is the correct way to include this lib?
Probably move the jar to libs folder and import it from there.

Comment (aggarwalpulkit596) [reply app/build.gradle]: @the-dagger bhaiya it is already in the libs folder there is some issue with the jetifier

**Pull request merged**